main.css

```diff
@@ -207,8 +207,8 @@ footer {
 #remainingDays {
   /* all:unset; */
   float: left;
-  margin-block: -0.5em 0.75em;
-  padding-left: 0.5em;
+  margin-block: -3.9em 0.75em;
+  padding-left: 12.5em;
```

Comment:
Why are you giving margins to the remaining days? Can't you place the remaining days in the label?

Reply:
Yes I can place it in label but still I will need to give margins and change some properties to it to place it to correct position.
Please tell which is more correct way? I will do it that way...